.github/workflows/cherry-pick-master-to-v6.yml

```diff
@@ -0,0 +1,34 @@
+name: Cherry pick master to v6
+
+on:
+  pull_request_target:
+    branches:
+      - master
+    types: ['closed']
+
+permissions: {}
+
+jobs:
+  cherry_pick_to_v6:
+    runs-on: ubuntu-latest
+    name: Cherry pick into v6
+    permissions:
+      pull-requests: write
+      contents: write
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'needs cherry-pick') && contains(github.event.pull_request.labels.*.name, 'v6.x') && github.event.pull_request.merged == true }}
```

Comment:
do we really need the `v6.x` label? Seems like 99.9% of PRs will be on `master` and when a cherry pick is needed we only add it to v6 anyways, right?

Reply:
IMHO I would be in favor of that

Reply:
Removed the extra check. 😉